src/js/components/purchaseOrder/PurchaseOrderStatus.jsx

```diff
@@ -30,17 +31,7 @@ const PurchaseOrderStatus = ({ status, allStatuses, fetchStatuses }) => {
     findStatusCircle(status);
   }, [allStatuses, status]);
 
-  // Capitalize only first letter of status
-  const statusToDisplay = status.charAt(0) + status.toLowerCase().slice(1);
-
-  return (
-    <div className="d-flex align-items-center justify-content-between">
-      <div className="d-flex justify-content-between align-items-center">
-        <span className={`${circle}-circle status-circle`} />
-        <span className="px-1">{statusToDisplay}</span>
-      </div>
-    </div>
-  );
+  return (<StatusIndicator status={status} variant={circle} />);
```

Comment:
maybe it's good idea to include this also in some short `.md` documentation if you have some time?